hack/check-links.sh

```diff
@@ -7,5 +7,10 @@ source ./hack/lib/common.sh
 header_text "Building the site and checking links"
 docker volume create sdk-html
 docker run --rm -v "$(pwd):/src" -v sdk-html:/src/website/public klakegg/hugo:0.73.0-ext-ubuntu -s website
-docker run --rm -v sdk-html:/target mtlynch/htmlproofer /target --empty-alt-ignore --http-status-ignore 429 --allow_hash_href
-docker volume rm sdk-html
+# more info: https://github.com/gjtorikian/html-proofer#special-cases-for-the-command-line
+docker run --rm -v sdk-html:/target mtlynch/htmlproofer /target \
+  --empty-alt-ignore \
+  --http-status-ignore 429 \
+  --allow_hash_href \
+  --url-ignore "/github.com\/operator-framework\/operator-sdk\/edit\/master\//"
+docker volume rm sdk-html
```

Comment:
#4205 has these changes now

Reply:
Approved the #4205. If we merge the #4205 first I will rebase it with the master. Otherwise, the #4205 can be rebased as well. I am looking for the review/approval in. this one indeed for we are able to push a follow up with the subsequent upstream changes. Could you please give a hand to use in its review?